stm/vio_checking.ml

```diff
@@ -141,6 +142,12 @@ let schedule_vio_compilation j fs =
     Worker.kill (Pool.find pid !pool);
     pool := Pool.remove pid !pool;
   done;
+  if !rc = 0 then begin
```

Comment:
If there's an error do we end up writing some random successful vos? If so then shouldn't we still update their times?

Reply:
That is the faster review ever ;-)
This code does exactly what the makefile line was doing.

I frankly don't know how to improve things. Removing some .vo files that were successfully built seems to bring nothing. But I don't get how setting the times of some .vo could help. I mean if you then manage to obtain the remaining vos, theirs date will differ anyway, confusing make.

So unless you have some other argument, I'd stay like that.

Reply:
>But I don't get how setting the times of some .vo could help

If a prefix succeeds it might make sense (eg everything in Init/ succeeds but somewhere else fails, then `make states` can tell nothing needs to be done)
Not a huge gain though